odbc/src/api/functions.rs

```diff
@@ -2824,9 +2824,23 @@ unsafe fn sql_get_stmt_attrw_helper(
                 SqlReturn::SUCCESS
             }
             StatementAttribute::SQL_ATTR_METADATA_ID => {
-                todo!();
+                // False means that we treat arguments to catalog functions as case sensitive. This
+                // is a _requirement_ for mongodb where FOO and foo are distinct database names.
+                *(value_ptr as *mut ULen) = SqlBool::False as ULen;
+                SqlReturn::SUCCESS
             }
-            _ => {
+            // leave SQL_GET_BOOKMARK as unsupported since it is for ODBC < 3.0 drivers
+            StatementAttribute::SQL_GET_BOOKMARK => {
+                err = Some(ODBCError::UnsupportedStatementAttribute(
+                    statement_attribute_to_string(attribute),
+                ));
+                SqlReturn::ERROR
+            }
+            // Not supported but still relevent to 3.0 drivers
+            StatementAttribute::SQL_ATTR_SAMPLE_SIZE
```

Comment:
```suggestion
            // leave SQL_GET_BOOKMARK as unsupported since it is for ODBC < 3.0 drivers
            StatementAttribute::SQL_GET_BOOKMARK 
            // Not supported but still relevent to 3.0 drivers
            | StatementAttribute::SQL_ATTR_SAMPLE_SIZE
```
We could still combine these into one set and have the comment differentiate them. No need to duplicate the `err = ...; SqlReturn::ERROR` code.